_haikus/demonic_sight_haiku.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Demonic Sight
+author: cyberd3mon
+---
+
+Demonic eye sees
+The outreach of digital
```

Comment:
Can you add a `<br>` tag or two spaces `  ` to the end of this line so that a line break is forced?